src/Resources/config/config.yml

```diff
@@ -8,11 +8,14 @@ knp_snappy:
     pdf:
         enabled: true
 
-prooph_service_bus:
-    event_buses:
-        sylius_invoicing_event_bus: ~
-    command_buses:
-        sylius_invoicing_command_bus: ~
+framework:
+    messenger:
+        default_bus: sylius_invoicing_plugin.command_bus
```

Comment:
I would extract it to test configuration file as [it has been done in RefundPlugin](https://github.com/Sylius/RefundPlugin/pull/89/files#diff-3cb4d392b9fb5e2d3cf5e66c4b081c7fR3)